server/test/test_colors.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+import anndata
+from server.common.colors import convert_color_to_hex_format, convert_anndata_category_colors_to_cxg_category_colors
+from server.test import PROJECT_ROOT
+from server.test.test_datasets.fixtures import pbmc3k_colors
+
+
+class ColorsTest(unittest.TestCase):
+    """ Test color helper functions """
+
+    def test_convert_color_to_hex_format(self):
```

Comment:
this should test bad format handling, ie,
* unknown names
* malformed hex
etc

and that the correct exceptions are thrown